---
title: data
disableToc: true
namespace: data
description: ''
dependencies: []
---



<div id="description" class="section level2">
<h2>Description</h2>
<ul>
<li>This directory contains data obtained from Clinvar.</li>
<li>The data is stored in parquet format. Descriptions for columns for each of the clinvar data tables are listed below. Each data table has unique column names.</li>
<li>The data was obtained from the following link <a href="https://ftp.ncbi.nlm.nih.gov/pub/clinvar/tab_delimited/special_requests/clinical_significance_updates.txt" class="uri">https://ftp.ncbi.nlm.nih.gov/pub/clinvar/tab_delimited/special_requests/clinical_significance_updates.txt</a></li>
</ul>
<div id="data-table-list" class="section level4">
<h4>Data Table List:</h4>
<ul>
<li><code>data/ACMG_gene_interp_history.parquet</code><br />
</li>
<li><code>data/allele_gene.txt.parquet</code></li>
<li><code>data/combined_variant_summary_xab.txt.parquet</code></li>
<li><code>data/combined_variant_summary_xac.txt.parquet</code></li>
<li><code>data/gene_specific_summary.parquet</code></li>
<li><code>data/organization_summary.parquet</code></li>
<li><code>data/variation_allele.txt.parquet</code></li>
<li><code>data/HGVS_summary.txt.parquet</code></li>
<li><code>data/clinical_significance_updates.parquet</code></li>
<li><code>data/hgvs4variation.txt.parquet</code></li>
<li><code>data/submission_summary.txt.parquet</code></li>
<li><code>data/cross_references.parquet</code><br />
</li>
<li><code>data/molecular_consequences.parquet</code></li>
<li><code>data/var_citations.parquet</code></li>
</ul>
</div>
<div id="gene_specific_summary" class="section level3">
<h3>gene_specific_summary</h3>
<ul>
<li><strong>Overview</strong> Shows the gene list,the number of submissions and the different alleles
<ul>
<li><em>Symbol</em>: Gene symbol (if officially named, from HGNC, else from NCBI's Gene database)</li>
<li><em>GeneID</em>: Unique identifier from NCBI's Gene database</li>
<li><em>Total_submissions</em>: Total submissions to ClinVar with variants in/overlapping this gene</li>
<li><em>Total_alleles</em> : Number of alleles submitted to ClinVar for this gene</li>
<li><em>Submissions_reporting_this_gene</em>: Subset of the total submissions that also reported the gene</li>
<li><em>Alleles_reported_Pathogenic_Likely_pathogenic</em>: Number of variants reported as pathogenic or likely pathogenic. Excludes structural variants that may overlap a gene</li>
<li><em>Gene_MIM_Number</em>: The MIM number for this gene</li>
<li><em>Number_Uncertain</em>: Submissions with an interpretation of 'Uncertain significance'</li>
<li><em>Number_with_conflicts</em>: Number of VariationIDs for this gene with conflicting interpretations ### variant_summary ( Located in <code>data/variant_summary/</code> <em>Note: These files are split into file chunks due to size</em>. Files are named with prefix <code>combined_variant_</code></li>
</ul></li>
<li>Contains variant information and the genomic position for each variant submitted to clinvar.</li>
<li>Variants have coordinates listed for hg19 and hg38</li>
<li>VariationID and AlleleID information per variant can be found in the <code>data/clinvar_variant_data/</code> and <code>data/clinvar_hgvs_data</code></li>
<li>This data is split into ~ 3 file chunks due to size. Each separate file has it's own header</li>
</ul>
</div>
<div id="cross_references" class="section level3">
<h3>cross_references</h3>
<ul>
<li><strong>Overview</strong> Generated on a weekly basis and indicates supports for clinical significance of variants located in other databases.</li>
<li><em>AlleleID</em>: integer value as stored in the AlleleID field in ClinVar (//<a href="mailto:Measure/@ID">Measure/@ID</a> in the XML)</li>
<li><em>Database</em>: name of the database</li>
<li><em>ID</em> : identifier used by that database</li>
<li><em>last_updated</em>: date the identifier /AlleleID relationship was created or last updated</li>
</ul>
</div>
<div id="var_citations" class="section level3">
<h3>var_citations</h3>
<ul>
<li>Weekly generated report of citations associated with data in clinvar AlleleID integer value as stored in the AlleleID field in ClinVar (//<a href="mailto:Measure/@ID">Measure/@ID</a> in the XML)</li>
<li><em>VariationID</em>: The identifier ClinVar uses to anchor its default display. (in the XML, //<a href="mailto:MeasureSet/@ID">MeasureSet/@ID</a>)</li>
<li><em>rs</em>: rs identifier from dbSNP, null if missing</li>
<li><em>nsv</em>: nsv identifier from dbVar, null if missing</li>
<li><em>citation_source</em>: The source of the citation, either PubMed, PubMedCentral, or the NCBI Bookshelf</li>
<li><em>citation_id</em>: The identifier used by that source</li>
</ul>
</div>
<div id="summary_of_conflicting_interpretations.txt" class="section level3">
<h3>summary_of_conflicting_interpretations.txt</h3>
<ul>
<li>Reports conflicting interpretations with respect to the variant. Doesn't report conflicting interpretations with respect to the phenotype.</li>
<li><em>Gene_Symbol</em>: If in a gene, its symbol</li>
<li><em>NCBI_Variation_ID</em>: The identifier ClinVar uses to anchor its default display. (in the XML, //<a href="mailto:MeasureSet/@ID">MeasureSet/@ID</a>)</li>
<li><em>ClinVar_Preferred</em> The preferred description ClinVar uses for this VariationID â”‚</li>
<li><em>Submitter1</em> Name of this submitter</li>
<li><em>Submitter1_SCV</em> Accession assigned to this submission</li>
<li><em>Submitter1_ClinSig</em> Clinical signficance asserted by this submitter</li>
<li><em>Submitter1_LastEval</em> Date last evaluated by this submitter</li>
<li><em>Submitter1_ReviewStatus</em> : Review status of this submission</li>
<li><em>Submitter1_Sub_Condition</em>: Submitted name of condition</li>
<li><em>Submitter1_Description</em>: Description of the interpretation</li>
<li><em>Submitter2</em>: Name of this submitter</li>
<li><em>Submitter2_SCV</em>: Accession assigned to this submission</li>
<li><em>Submitter2_ClinSig</em>: Clinical significance asserted by this submitter</li>
<li><em>Submitter2_LastEval</em>: Date last evaluated by this submitter</li>
<li><em>Submitter2_ReviewStatus</em>: Review status of this submission</li>
<li><em>Submitter2_Sub_Condition</em>: Submitted name of condition</li>
<li><em>Submitter2_Description</em>: Description of the interpretation</li>
<li><em>Rank_diff</em>: Rank value assigned to the differences in interpretation: -1: one of the interpretations is not in the set of Pathogenic; Pathogenic, low penetrance; Established risk allele;Likely pathogenic; Likely pathogenic, low penetrance; Likely risk allele; Uncertain significance; Uncertain risk allele; Likely benign; Benign 1-4, difference when both interpretations are in the set of Pathogenic; Pathogenic, low penetrance; Established risk allele; Likely pathogenic; Likely pathogenic, low penetrance; Likely risk allele; Uncertain significance;Uncertain risk allele; Likely benign; Benign</li>
<li><em>Conflict_Reported</em>: yes or no. Useful to supplement the Rank_diff column when Rank_diff = 1 but a conflict is still reported.</li>
<li><em>Variant_Type</em>: the type of variant being described</li>
<li><em>Submitter1_Method</em>: the collection method(s) reported by this submitter</li>
<li><em>Submitter2_Method</em>: the collection method(s) reported by this submitter</li>
</ul>
</div>
<div id="hgvs4variation.txt.gz" class="section level3">
<h3>hgvs4variation.txt.gz</h3>
<ul>
<li>hgvs expressions reported by ClinVar based on variation and allele IDs</li>
<li>Column information is included in the file header</li>
<li><em>GeneSymbol</em>: the offical symbol from HGNC for the gene in which this allele is found. Will be null if the variant is intergenic or spans multiple genes.</li>
<li><em>GeneID</em>: NCBI's identfier for the gene in which this allele is found. Will be null if the variant is intergenic or spans multiple genes.</li>
<li><em>VariationID</em>: the identifier assigned by ClinVar and used to build the URL, namely <a href="https://ncbi.nlm.nih.gov/clinvar/VariantID" class="uri">https://ncbi.nlm.nih.gov/clinvar/VariantID</a></li>
<li><em>AlleleID</em>: the identifier assigned by ClinVar to the simple allele. Will be null if the allele is complex.</li>
<li><em>Type</em>: the type of HGVS expression</li>
<li><em>Assembly</em>: reported only for assembly-specific HGVS expressions</li>
<li><em>NucleotideExpression</em>: an HGVS expression based on a nucleotide sequence; may be null if the allele is defined only by a protein change</li>
<li><em>NucleotideChange</em>: the subset of the NucleotideExpression in the previous column representing the sequence change</li>
<li><em>ProteinExpression</em>: an HGVS expression based on a protein sequence; if NucleotideExpression is provided, then represents the translation of nucleotide change</li>
<li><em>ProteinChange</em>: the subset of the ProteinExpression in the previous column representing the sequence change</li>
<li><em>UsedForNaming</em>: the accession in this HGVS expression is used to report a name in ClinVar. Values are Yes or No.</li>
<li><em>Submitted</em>: this HGVS expression was used by a submitter. Values are Current or Previous (provided in a previous version of a submission (SCV accession) but not in a current SCV or current version of an SCV)</li>
<li><em>OnRefSeqGene</em> : the accession in this HGVS expression is annotated on a RefSeqGene</li>
</ul>
</div>
<div id="submission_summary" class="section level3">
<h3>Submission_summary</h3>
<ul>
<li>Overview of phenotypes, observations and interpretations</li>
<li>Table includes VariationID and ClinicalSignificance</li>
</ul>
<ol style="list-style-type: decimal">
<li><em>VariationID</em>: the identifier assigned by ClinVar and used to build the URL, namely <a href="https://ncbi.nlm.nih.gov/clinvar/VariationID" class="uri">https://ncbi.nlm.nih.gov/clinvar/VariationID</a></li>
<li><em>ClinicalSignificance</em>: interpretation of the variation-condition relationship</li>
<li><em>DateLastEvaluated</em>: the last date the variation-condition relationship was evaluated by this submitter</li>
<li><em>Description</em>: an optional free text description of the basis of the interpretation</li>
<li><em>SubmittedPhenotypeInfo</em>: the name(s) or identifier(s) submitted for the condition that was interpreted relative to the variant</li>
<li><em>ReportedPhenotypeInfo</em>: the MedGen identifier/name combinations ClinVar uses to report the condition that was interpreted.'na' means there is no public identifier in MedGen for the condition.</li>
<li><em>ReviewStatus</em>: the level of review for this submission, namely <a href="http://www.ncbi.nlm.nih.gov/clinvar/docs/variation_report/#review_status" class="uri">http://www.ncbi.nlm.nih.gov/clinvar/docs/variation_report/#review_status</a></li>
<li><em>CollectionMethod</em>: the method by which the submitter obtained the information provided</li>
<li><em>OriginCounts</em>: the reported origin and the number of observations for each origin</li>
<li><em>Submitter</em>: the submitter of this record</li>
<li><em>SCV</em>: the accession and current version assigned by ClinVar to the submitted interpretation of the variation-condition relationship</li>
<li><em>SubmittedGeneSymbol</em>: the symbol provided by the submitter for the gene affected by the variant. May be null.</li>
<li><em>ExplanationOfInterpretation</em>: the submitter's preferred term for the interpretation when ClinicalSignificance is submitted as 'other' or 'drug response'. May be null.</li>
</ol>
</div>
<div id="acmg_gene_interp_history" class="section level3">
<h3>ACMG_gene_interp_history</h3>
<ul>
<li>GeneSymbol</li>
<li>Accession</li>
<li>Intepretation</li>
<li>Date last evaluated</li>
<li>Whether this intepretation is current or previous ### Allele_gene AlleleID: the integer identifier assigned by ClinVar to each simple allele GeneID: integer, GeneID in NCBI's Gene database<br />
Symbol: character, Symbol preferred in NCBI's Gene database. Is the symbol from HGNC when available Name: character, full name of the gene GenesPerAlleleID: integer, number of genes related to the allele Category: character, type of allele-gene relationship Source: character, was the relationship submitted or calculated? ### Organization_summary organization : the name of the lab and the institution of which it is part<br />
organization ID: the id used in ClinVar and GTR; often reported as OrgID; append to <a href="https://www.ncbi.nlm.nih.gov/clinvar/submitters" class="uri">https://www.ncbi.nlm.nih.gov/clinvar/submitters</a> to review more details institution type: type of organization street address: street address country: country number of ClinVar submissions: number of submission to ClinVar date last submitted: last date on a public submission from this organization maximum review status: the 'most stars' valid for any submission from this organization collection methods: comma-delimited list of methods used to determine information for the submission novel and updates :values are novel, novel and updates. The latter indicates the submitter has provided updates. clinical significance categories submitted: list of types of interpretations from this organization number of submissions from clinical testing: number of submissions for the list of categories in 'collection methods' number of submissions from research: number of submissions for the list of categories in 'collection methods' number of submissions from literature only: number of submissions for the list of categories in 'collection methods' number of submissions from curation: number of submissions for the list of categories in 'collection methods' number of submissions from phenotyping: number of submissions for the list of categories in 'collection methods' ### variation_allele Mapping of ClinVar's VariationID (used to build the URL on the web site) and the AlleleIDs assigned to each simple variant.</li>
</ul>
<ol style="list-style-type: decimal">
<li>VariationID: the identifier assigned by ClinVar and used to build the URL, namely <a href="https://ncbi.nlm.nih.gov/clinvar/VariationID" class="uri">https://ncbi.nlm.nih.gov/clinvar/VariationID</a></li>
<li>Type: Types of VariationID include Variant (simple variant), Haplotype, CompoundHeterozygote, Complex, Phase unknown, Distinct chromosomes</li>
<li>AlleleID: the integer identifier assigned by ClinVar to each simple allele</li>
<li>Interpreted: yes_ indicates an interpretation was submitted about the VariationID specifically, <em>no</em> indicates that information about the VariationID was submitted as a component of a different record. ### HGVS_summary</li>
</ol>
<ul>
<li>A tab-delimited file of HGVS transcript expressions based on RefSeqs</li>
<li><em>NCBI</em>: GeneID</li>
<li><em>HGNC</em>: Gene symbol</li>
<li><em>ClinVar VariationID</em>: (What is used to build the URL)</li>
<li><em>ClinVar AlleleID</em> : (The identifier for a simple variant)</li>
<li><em>HGVS coding</em></li>
<li><em>coding change</em></li>
<li><em>HGVS protein</em></li>
<li><em>protein change</em> ### clinical_significance_updates mset_id: set ID variant_name: Based on RefSeq and variant coordinates<br />
previous_value: Indicates previous prediction of level of pathogenicity/benign/Uncertain significance<br />
current_value: Indicates current prediction of level of pathogenicity/benign/Uncertain significance date_modified ### molecular_consequences This file contains the HGVS expression, the Sequence Ontology identifier, and the value calculated for the molecular consequence of that HGVS expression. ## Data Retrieval</li>
<li>Data on the ftp website are updated on a weekly/monthly basis. <a href="https://ftp.ncbi.nlm.nih.gov/pub/clinvar/tab_delimited" class="uri">https://ftp.ncbi.nlm.nih.gov/pub/clinvar/tab_delimited</a></li>
</ul>
</div>
<div id="data-stored-in-parquet-files" class="section level3">
<h3>Data stored in Parquet files</h3>
<ul>
<li>Data can be downloaded using the following commands. To retrieve the data, make sure that dvc is downloaded</li>
</ul>
<p><strong>Retrieving a single file</strong></p>
<pre><code>dvc get git@github.com:insilica/oncindex-bricks.git bricks/clinvar/data/allele_gene.txt.parquet -o data/allele_gene.txt.parquet</code></pre>
<p><strong>Retrieving a folder</strong></p>
<pre><code>dvc get git@github.com:insilica/oncindex-bricks.git bricks/clinvar/data/variant_summary -o data/variant_summary</code></pre>
<p><strong>It is advised to import files into a project so that you will be able to track changes in the data set</strong></p>
<pre><code>dvc import git@github.com:insilica/oncindex-bricks.git bricks/clinvar/data/variant_summary -o data/variant_summary</code></pre>
</div>
</div>
